apollo-configservice/src/main/java/com/ctrip/framework/apollo/configservice/wrapper/DeferredResultWrapper.java

```diff
@@ -64,4 +65,9 @@ public void setResult(List<ApolloConfigNotification> notifications) {
   public DeferredResult<ResponseEntity<List<ApolloConfigNotification>>> getResult() {
     return result;
   }
+
+  @Override
+  public int compareTo(@NonNull DeferredResultWrapper deferredResultWrapper) {
+    return Integer.compare(this.hashCode() - deferredResultWrapper.hashCode(), 0);
```

Comment:
I think this is ok, but is there any difference between `return Integer.compare(this.hashCode(), deferredResultWrapper.hashCode())`?